src/install-edm-linux.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```

Comment:
nit : can we add a copyright header here? The same applies for the other `install-edm-*` shell and cmd scripts.